apps/condo/domains/user/utils/redisGuard.js

```diff
@@ -0,0 +1,55 @@
+const conf = require('@core/config')
+const IORedis = require('ioredis')
+// TODO(zuch) : remove WORKER_REDIS_URL when .env will be updated
+const REDIS_URL = conf['REDIS_URL'] || conf['WORKER_REDIS_URL']
```

Comment:
Just a question, does `WORKER_REDIS_URL` == `DOCKER_COMPOSE_WORKER_REDIS_URL`?

Reply:
Please don’t use worker redis :) for guards

Reply:
Done